src/trace/trace.service.ts

```diff
@@ -68,26 +79,42 @@ export async function getTrace({
   return toDto({ trace, mlflowTrace, flags: query });
 }
 
-export async function createTrace(traceBody: TracePostBody): Promise<TraceDto> {
-  const traceId = new ObjectId().toString();
+export async function createTrace(traceBody: ExportTraceServiceRequest__Output): Promise<TraceDto> {
+  const spans = [...traceBody.resourceSpans].flatMap((resourceSpan) => {
```

Comment:
I think it would also be good to check the instrumentation scope as a guardrail for unwanted data?

Reply:
done in https://github.com/i-am-bee/bee-observe/pull/7/commits/f1485c7eb8bad1995168b42304c9c3a858e904a7

Reply:
Why the filter, wouldn't you rather fail if you get invalid data?

Reply:
No, in case we send more data scopes on observe, it should be able to filter only the right one.